### DISCUSSION TITLE
Conflito no tipo de operando: datetime é incompatível com text - (Curso 2811) Autenticação e Autorização

### DISCUSSION THREAD

**Discussion by MateusDaviDev** [balta-io/forum#408] [Tira Dúvidas] [answered]
Msg 206, Level 16, State 2, Line 24
Conflito no tipo de operando: datetime é incompatível com text
Msg 206, Level 16, State 2, Line 25
Conflito no tipo de operando: datetime é incompatível com text
Total execution time: 00:00:00.008

Aconteu esse erro ao executar o script sql ( https://github.com/balta-io/2811/blob/main/Modulo%203/Scripts/blog.sql )

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MateusDaviDev, como vai?

Provavelmente você está tentando inserir um texto em algum campo datetime...
Como estão os campos das tabelas que está tentando inserir os dados? Quais colunas e quais tipos destas colunas?
